src/miniscript/context.rs

```diff
@@ -53,6 +56,11 @@ impl fmt::Display for ScriptContextError {
                 "At least one spending path in the Miniscript fragment has more \
                  witness items than MAX_STANDARD_P2WSH_STACK_ITEMS.",
             ),
+            ScriptContextError::MaxOpCountExceeded => write!(
+                f,
+                "At least one satisfaction path in the Miniscript fragment contains \
+                 more than `MAX_OPS_PER_SCRIPT`(201) opcodes."
```

Comment:
nit: use variable MAX_OPS_PER_SCRIPT variable in format string instead of hardcoded 201.

Reply:
Or you can remove 201 completely. Anything works

Reply:
I don't expect this value to change much anyways :p
That said, removed it and the backticks from the error message as it makes it easier to match errors against it.